websites/R/Raven Code/dist/metadata.json

```diff
@@ -1,23 +1,19 @@
 {
 	"$schema": "https://schemas.premid.app/metadata/1.7",
 	"author": {
-		"name": "can",
-		"id": "613700645173592086"
+		"name": "skyhan",
+		"id": "949705281863032872"
```

Comment:
Have you been in contact with the old presence author to authorise this change? If you haven't got permission, please add yourself as a contributor instead

Reply:
That's my account. It was disabled for some reason I don't understand.